.golangci.yml

```diff
@@ -62,15 +62,10 @@ run:
 #    - sample
 #  skip-files:
 #    - sample
-
 issues:
-  # Show only new issues: if there are unstaged changes or untracked files,
-  # only those changes are analyzed, else only changes in HEAD~ are analyzed.
-  # It's a super-useful option for integration of golangci-lint into existing large codebase.
-  # It's not practical to fix all existing issues at the moment of integration:
-  # much better don't allow issues in new code.
-  #
-  # Default: false.
-  new: true
-  # Show only new issues created after git revision `REV`.
-  new-from-rev: HEAD
+  exclude-rules:
+    - path: test/
+      linters:
+        - gosec
+        - lll
+        - unparam
```

Comment:
gosec - we have variable names with "password" and "secret" in there names.
lll - for big lines where readability 
unparam -  for those general functions which can be used in future with different argument value.
should I mark these nolint inline?

Reply:
Yeah, inline would be right way to do otherwise this exclude lint might ignore other genuine issues.
For long msg comment length, we can add specific ignore lll with regex